spec/identity/hostdata_spec.rb

```diff
@@ -336,7 +348,11 @@
 
   describe '.secrets_s3' do
     before do
+      stub_request(:put, 'http://169.254.169.254/latest/api/token').
+        with(headers: { 'X-Aws-Ec2-Metadata-Token-Ttl-Seconds' => '60' }).
+        to_return(body: "bhasdkjhas82")
```

Comment:
I wonder if we should leave a note here that `to_return(body: "bhasdkjhas82")` needs to match the token value in the stubbed request below. That was not easy to debug.

Reply:
perhaps a note in GH will suffice

Reply:
we can replace these with a variable so they match?

Reply:
PR: https://github.com/18F/identity-hostdata/pull/30